src/main/java/org/jenkinsci/plugins/ewm/definitions/Disk.java

```diff
@@ -11,8 +11,7 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
 
-import static hudson.Util.fixEmptyAndTrim;
-import static hudson.Util.isRelativePath;
+import static hudson.Util.*;
```

Comment:
It's better to avoid doing it in the production code, because dependency updates may lead to the conflicts in the code